app/src/components/features/rules/RuleBuilder/Header/ActionButtons/actions/index.js

```diff
@@ -52,7 +54,10 @@ export const saveRule = async (appMode, ruleObject, callback) => {
       exit();
     })
     .catch(() => {
-      toast.error("Error in saving rule. Please contact support.");
+      Logger.log("Error in saving rule");
+      trackErrorInRuleCreation("save_rule_error", ruleToSave.ruleType);
+      Sentry.captureException(new Error("Extension: Error in saving rule", "fatal"));
```

Comment:
log the actual error as well, so that we can know while debugging if it timed out or was there a bug